cloudify_agent/installer/__init__.py

```diff
@@ -114,7 +115,9 @@ def _from_package(self):
         self.logger.info('Untaring Agent package...')
         self.extract(archive=package_path,
                      destination=self.cloudify_agent['agent_dir'])
-        configure = '--relocated-env'
+        configure = ''
+        if platform.system() == 'Linux':
+            configure = '--relocated-env'
```

Comment:
this runs on manager, use self.cloudify_agent['windows'] to figure out 

also, requiretty should be inside of the if/else